examples/workflows/hexacene.xyz

```diff
@@ -0,0 +1,44 @@
+42
+Lattice="43.60106 0.0 0.0 0.0 20.108226 0.0 0.0 0.0 10.0" Properties=species:S:1:pos:R:3:masses:R:1 pbc="F F F"
```

Comment:
I am not sure this structure can be run on 1 CPU. In case it can't, would you be able to provide a smaller one?

Reply:
good point, I will add a smaller one